src/main/java/uk/gov/hmcts/divorce/ciccase/model/ApplicantCICDetails.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.divorce.ciccase.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.ccd.sdk.api.CCD;
+import uk.gov.hmcts.ccd.sdk.type.AddressGlobalUK;
+import uk.gov.hmcts.ccd.sdk.type.YesOrNo;
+import uk.gov.hmcts.divorce.ciccase.model.access.CaseworkerWithCAAAccess;
+import uk.gov.hmcts.divorce.ciccase.model.access.DefaultAccess;
+
+import java.time.LocalDate;
+
+import static uk.gov.hmcts.ccd.sdk.type.FieldType.Email;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonNaming(PropertyNamingStrategies.UpperCamelCaseStrategy.class)
+@Builder
+public class ApplicantCICDetails {
+
+
+    @CCD(label = "Is represented by a solicitor?")
+    private YesOrNo solicitorRepresented;
+    @CCD(
```

Comment:
Minor quibble, but should we begin a flag name with "is" viz. isSolicitorRepresented ?

Reply:
agree